iio/iio_app/iio_app.c

```diff
@@ -122,6 +111,7 @@ static inline uint32_t _calc_uart_xfer_time(uint32_t len, uint32_t baudrate)
 #if !defined(LINUX_PLATFORM) && !defined(NO_OS_NETWORKING)
 static int32_t iio_print_uart_info_message(struct no_os_uart_desc **uart_desc,
 		struct no_os_uart_init_param *uart_init_par,
+		struct no_os_uart_init_param *user_uart_params,
```

Comment:
it doesn't make sense to have 2 uart_init_params!

this function only needs 1, the one from the user

the other one (with the defaults) you can create internally with hardcoded values (115200,8n1n)

Reply:
Only one uart_init_params used.